src/web-ui/src/partials/TextAlerts/TextAlerts.vue

```diff
@@ -1,17 +1,31 @@
 <template>
   <section v-if="isEnabled && user" class="section container p-4">
     <h1 class="heading mb-1">Join <span class="text-alerts">text alerts</span> and get 20% off</h1>
-    <p class="disclaimer mb-2">Message and data rates may apply. See details.</p>
-    <form @submit.prevent="onSubmit" class="mb-2 form d-flex justify-content-center align-items-stretch">
-      <input
-        type="tel"
-        name="phoneNumber"
-        placeholder="Enter cellphone number"
-        v-model="phoneNumber"
-        v-mask="'+1 (###) ### - ####'"
-        class="input py-1 px-2"
-      />
-      <button type="submit" class="submit btn">Submit</button>
+    <p class="mb-3">
+      Enter your mobile number to receive texts about the Retail Demo Store, including Amazon Pinpoint.
+    </p>
+    <form @submit.prevent="onSubmit" class="mb-2 form">
+      <div class="mb-2 d-flex justify-content-center align-items-stretch">
+        <input
+          type="tel"
+          name="phoneNumber"
+          placeholder="Enter cellphone number"
+          v-model="phoneNumber"
+          v-mask="'+1 (###) ### - ####'"
+          class="input py-1 px-2"
+        />
+
+        <button type="submit" class="submit btn" :disabled="!hasConsented">Submit</button>
+      </div>
+
+      <div class="consent d-flex align-items-start text-left">
+        <input type="checkbox" class="consent-checkbox mr-2" id="text-alerts-consent" v-model="hasConsented" />
+        <label class="" for="text-alerts-consent"
+          >I consent to receive automated text messages (including marketing messages) from or on behalf of Amazon Web
+          Services about the Retail Demo Store, including Amazon Pinpoint, at my mobile number above. Consent is not a
+          condition of any purchase. Message and data rates may apply.</label
```

Comment:
Please add in the text around on how the data entered will only be used for demonstrating features of messaging/email within the demo experience

Reply:
Done